backend/tracim_backend/lib/translate/translator.py

```diff
@@ -0,0 +1,78 @@
+from abc import ABC
+from abc import abstractmethod
+from collections import namedtuple
+from typing import BinaryIO
+from typing import List
+
+AUTODETECT_LANG = "auto"
+TranslationLanguagePair = namedtuple("TranslationLanguagePair", ["input_lang", "output_lang"])
```

Comment:
We can't get rid of Python 3.5 soon enough... it would allow using typing.NamedTuple or dataclasses.